mainwindow.cpp

```diff
@@ -0,0 +1,15 @@
+#include "mainwindow.h"
+#include "./ui_mainwindow.h"
+
+MainWindow::MainWindow(QWidget *parent)
+    : QMainWindow(parent)
+    , ui(new Ui::MainWindow)
+{
+    ui->setupUi(this);
```

Comment:
What is setupUI? I don't see reference to this in the header or various cpp files. 

Also, the naming convention I have been using is Pascal for classes, snake for functions/variables/file names. I am not opposed to changing convention, just want to make sure we are remaining consistent